packages/web-app-admin-settings/src/components/Spaces/SpacesList.vue

```diff
@@ -185,12 +186,14 @@ export default defineComponent({
             b = getTotalQuota(s2).toString()
```

Comment:
Same solution needs to be applied to `totalQuota` sorting, right?

Reply:
Oh ya, missed that